packages/storage-adapter-local-forage/README.md

```diff
@@ -0,0 +1,63 @@
+# index
+
```

Comment:
rebase off of master; the change to stop generating readme, so this can be replaced with somethign meaningful. (Take a look at the other readmes; they're all basically just "plugin name plus go to docs site")